pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -1386,6 +1381,18 @@ func (cf *changeFrontier) Start(ctx context.Context) {
 		return
 	}
 
+	err = checkpoint.Restore(cf.frontier, cf.spec.SpanLevelCheckpoint)
+
+	if err != nil {
```

Comment:
nit: replace with `if err := ...; err != nil {` and in general, don't put newlines between function calls that return an error and the subsequent err check